content/_includes/utility.macros.njk

```diff
@@ -232,9 +232,9 @@ params:
       {% set same = 'same-y' if (start_y == end_y) else none %}
       {% set same = 'same-m' if same and (start_m == end_m) else same %}
 
-      {{ start | getDate(ranges[same].start or 'month_year') }}
+      {{ start | getDate(ranges[same].start or 'year') }}
       {{- '--' | typogr | safe -}}
-      {{ end | getDate(ranges[same].end or 'month_year') }}
+      {{ end | getDate(ranges[same].end or 'year') }}
```

Comment:
I'm curious what the goal of this change was -- it has an impact throughout the site, because this macro is used so broadly. (Not saying it's a bad change throughout; but noticing that it does touch a lot of pages.)

Reply:
@jgerigmeyer I asked for that change because I didn't think the month start and end would be that useful to visitors and because I didn't want to have to spend the time figuring out which months we started and ended each of our projects. If it is causing problems elsewhere, though, I could be convinced to keep month_year.

Reply:
Sure, that makes sense. I'm not sure it's causing problems elsewhere, I just noticed that the change is global throughout many dates on the site.